ui/helpers/utils/transactions.util.js

```diff
@@ -71,18 +62,11 @@ export async function getMethodDataAsync(fourBytePrefix) {
       registry = new MethodRegistry({ provider: global.ethereumProvider });
```

Comment:
We can remove the registry entirely and the `eth-method-registry` dependency.

Reply:
We still use it to [parse the signature from 4byte](https://github.com/MetaMask/metamask-extension/pull/15300/files/cb31bfc23bae1e11c08c04a11a187938e80714a8#diff-c8c256cb0389d020fcf5c440c7133f063c3d2a1b6321b3252d23cd0208bb79dbL85). Are you suggesting we build a utility function to do this ourselves?

Reply:
I suppose we could just [copy over the `parse` method](https://github.com/MetaMask/eth-method-registry/blob/main/src/index.ts#L52) from `eth-method-registry` since we don't need anything else from the package here

Reply:
Ah, my mistake. Yeah we could move it out, but it doesn't have to be as part of this PR imo.